docs/layouts/partials/footer.html

```
@@ -1,165 +1,232 @@
-<footer class="footer">
-  <div class="text-white">&copy; {{ now.Year }} The Meshery Authors</div>
+<footer>
```

HTMLHint (1.9.2)

[error] Doctype must be declared before any non-comment content.

(doctype-first)